backend/app/Controllers/Http/ForgotPasswordController.js

```diff
@@ -0,0 +1,38 @@
+const { randomBytes } = require('crypto');
+const { promisify } = require('util');
+
+const Mail = use('Mail');
+const Env = use('Env');
+
+const User = use('App/Models/User');
+
+class ForgotPasswordController {
+  async store({ request }) {
```

Comment:
Por essa controller usar apenas a model de User, a função que ela executa sera que não faria sentido dentro da controller de User?
```suggestion
  /**
   * Generete stores and send to user a token for change password.
   * POST forgot
   *
   * @param {Request} ctx.request
   */
  async store({ request }) {
```